### ISSUE TITLE
Generate SQL code: setting for DROP existing/Do NOT DROP existing

### ISSUE THREAD

**Issue by BBergerac** [pgmodeler/pgmodeler#1740]
I found this tool by chance. The features are impressive but here's a suggestion for SQL generation:
Add a a **setting to choose between CREATE options**.
For example:
**DROP** TABLE IF EXISTS public.customers CASCADE;
and
**CREATE** TABLE public.customers ...
Currently, the DROP instruction is commented out which is already nice but if there are dozens of table, uncommenting might be too much work.
Additional issue: if a table already contains data DROP would delete it.
Maybe an **ALTER** instruction as another option?

And just a note: Your homepage says:
With its dynamic code generation, pgModeler can export the designed models to different versions of PostgreSQL (currently from 9.x to **13.x**).
But **PG 15** is also supported.

**Comment by rkhaotix** [comment]
Hi @BBergerac 

> I found this tool by chance. The features are impressive but here's a suggestion for SQL generation:
Add a a setting to choose between CREATE options.
For example:
DROP TABLE IF EXISTS public.customers CASCADE;
and
CREATE TABLE public.customers ...
Currently, the DROP instruction is commented out which is already nice but if there are dozens of table, uncommenting might be too much work.

If you use pgModeler to export you can check the option to drop the objects first before creating them:

![image](https://user-images.githubusercontent.com/2205476/217086228-a1df1c13-c868-465b-aa24-c2ed7261630c.png)

> Additional issue: if a table already contains data DROP would delete it.
Maybe an ALTER instruction as another option?

Tables containing data are always a problem to be considered, that's why I always recommend having a backup of your database before proceeding with any change proposed by pgModeler. For the ALTER command, you have the diff process at your disposal, maybe this is what you need, no?

https://www.pgmodeler.io/support/docs/53-compare-model-and-database?v=1.0.0

> And just a note: Your homepage says:
With its dynamic code generation, pgModeler can export the designed models to different versions of PostgreSQL (currently from 9.x to 13.x).
But PG 15 is also supported.

Fixed that, thanks!

**Comment by tengels** [comment]
This choice between dropping DB or Objects is only available for (direct) export to database is it not?

Does a similar option exist for the export to file? These currently do include the DROP statement, but always (?) as indicate in the first post commented out.

**Comment by rkhaotix** [comment]
@tengels yes, this option is only available in the direct export, but it's a great idea to make a switch to enable drop commands in the export to file too.
